docs/7-testing-guide/README.md

```diff
@@ -1,5 +1,6 @@
 # Testing guide
```

Comment:
I saw the issue in https://github.com/aragon/staking/issues/35, but I'm not sure there's much point in having this in the docs since there's not a lot of setup needed. Perhaps we should move this to be with the contract's README?